### PR TITLE
ERM-603 and ERM-604: PR to change the way organization cards are rendered

### PR DESCRIPTION
**Pull request by EthanFreestone**
This is a little bit of work to change the way that the name of an organization is rendered on an organization card.

I've had some problems getting my agreements form to work, so I've been unable to test whether all tests pass on ui-agreements, but there seem to be no issues in stripes-erm-components or ui-licenses using this, at least for me locally.

### CODE REVIEW

File: lib/ViewOrganizationCard/ViewOrganizationCard.js [hunk 1]
```diff
@@ -139,16 +141,16 @@ export default class ViewOrganizationCard extends React.Component {
...
     const { id } = this.props;
     return (
-      <span>
-        <AppIcon
-          app="organizations"
-          size="small"
-        >
-          <strong id={id} data-test-org-name>
-            {orgName}
+      <Col>
```
Comment (doytch) [inline lib/ViewOrganizationCard/ViewOrganizationCard.js]: Cols need to be children of Rows, not vice versa.

File: lib/ViewOrganizationCard/ViewOrganizationCard.js [hunk 2]
```diff
@@ -139,16 +141,16 @@ export default class ViewOrganizationCard extends React.Component {
...
+        <Row>
+          <strong>
+            Name
```
Comment (doytch) [inline lib/ViewOrganizationCard/ViewOrganizationCard.js]: This needs to be a translated string.

File: lib/ViewOrganizationCard/ViewOrganizationCard.js (hunk 1 above)
Comment (EthanFreestone) [reply lib/ViewOrganizationCard/ViewOrganizationCard.js]: Whoops, I've switched them now

File: lib/ViewOrganizationCard/ViewOrganizationCard.js (hunk 2 above)
Comment (EthanFreestone) [reply lib/ViewOrganizationCard/ViewOrganizationCard.js]: Yeah, I missed that one, cheers

**Pull request merged**